### DISCUSSION TITLE
Can't install --> "Failed setup: Modbus/TCP connect to 192.168.178.4:1502 failed"

### DISCUSSION THREAD

**Discussion by uhl8080** [WillCodeForCats/solaredge-modbus-multi#640] [General]
### Describe the bug

I have static IP address of my SE 17k, on WiFi network. With Energy meter. Ping work all the time. 
MODBUS is enabled on ID 1, port 1502. Of course, Solaredge cloud platform is accessible and works.
Have no idea how to test this issue, where to start etc. - maybe switch from Wifi to Ethernet port ?

### Expected behavior

Maybe more in-deep error description.

### Diagnostic File

![image](https://github.com/user-attachments/assets/1bec8019-9b12-4542-ba57-e69f9d281fca)

### Home Assistant Version

2024.7.3

### solaredge-modbus-multi Version

2.4.18

### Installation Type

HAOS

### Read the Instructions

- [X] My configuration follows https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki/Configuration
- [X] I have read the Known Issues https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki/Known-Issues
- [X] This issue is not about a template error (do not report template errors, read https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki/Template-Design-Notes)

### Additional Context

Btw: in search of solution i tried another Solaredge solution from @binsetsu "SOLAREDGE MODBUS TCP" and it starts immediately with same IP, with WiFi network, same MODBUS configuration, etc... !
So now i'm really confused. Then i reinstall HAOS and give another try to Solaredge Multi- again same error: Failed to connect..

**Comment by WillCodeForCats** [comment]
I don't know what the difference would be. Many people use this one successfully.

Remember only one connection to the inverter is allowed.

**Comment by Uhae** [comment]
Seems strange indeed. Maybe try a Modbus Proxy as a HA addon?

**Comment by seatrope** [comment]
Definitely try Ethernet. 

Maybe something is blocking on Wifi. I think there was discussion before that SE disabled Modbus connection through Wifi recently in firmware. That would be my bet.

**Comment by uhl8080** [comment]
In the meantime, I've switched the inverter's site communication from Wi-Fi to Ethernet, and that has really helped! Thank you for expressing your concern.

**Comment by devdems** [comment]
I'm facing the same issue and i'm on ethernet. It works with "SOLAREDGE MODBUS TCP" from binsetsu but not with this one.
